src/app/services/property-detail/property-detail.service.spec.ts

```diff
@@ -7,38 +7,51 @@ import { LocalStorageService } from 'src/app/services/local-storage.service';
 import { httpClientSpy, resetSpies } from 'src/app/helpers/spies';
 
 describe('PropertyDetailService', () => {
-    let propertyDetailService: PropertyDetailService;
-    beforeAll(() => resetSpies([httpClientSpy]));
-    afterAll(() => resetSpies([httpClientSpy]));
+  let propertyDetailService: PropertyDetailService;
+  beforeAll(() => resetSpies([httpClientSpy]));
+  afterAll(() => resetSpies([httpClientSpy]));
 
-    beforeEach(() => {
-        TestBed.configureTestingModule({
-            imports: [HttpClientTestingModule],
-            providers: [
-                LocalStorageService,
-                {
-                    provide: HttpClient, useValue: httpClientSpy
-                }
-            ]
-        });       
-        propertyDetailService = TestBed.get(PropertyDetailService);
-    }
-    );
-
-    it('should be created', () => {
-        propertyDetailService = TestBed.get(PropertyDetailService);
-        expect(propertyDetailService).toBeTruthy();
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule],
+      providers: [
+        LocalStorageService,
+        {
+          provide: HttpClient,
+          useValue: httpClientSpy
+        }
+      ]
     });
+    propertyDetailService = TestBed.get(PropertyDetailService);
+  });
+
+  it('should be created', () => {
+    propertyDetailService = TestBed.get(PropertyDetailService);
+    expect(propertyDetailService).toBeTruthy();
+  });
 
-    it('should test getting property', () => {
-        const slug = 'hello';  
-        const responseObject = {
-            success: true,
-            message: 'successful'
-        };
-        httpClientSpy.get.and.returnValue(of(responseObject));
-        propertyDetailService.getProperty(slug).subscribe(res => {
-            expect(res).toBe(responseObject);
-        });
+  it('should test getting property', () => {
+    const slug = 'hello';
+    const responseObject = {
+      success: true,
+      message: 'successful'
+    };
+    httpClientSpy.get.and.returnValue(of(responseObject));
+    propertyDetailService.getProperty(slug).subscribe(res => {
+      expect(res).toBe(responseObject);
+    });
+  });
+  it('should test display_Modal', () => {
+    propertyDetailService.display_Modal();
```

Comment:
We could add an assertion here in this test, since its currently just calling `display_Modal()`

Reply:
worked on that @Tevinthuku